### ISSUE TITLE
Specification name

### ISSUE THREAD

**Issue by SA-StatNL** [jdemetra/jdemetra-app#200]
It would be very helpful to see the name of the specification in a multi-document. Currently, only ‘X13’ is depicted, which is not clear and can lead to mistakes. Also in the output files, it would be useful to see which specification was used.

**Comment by palatej** [comment]
I agree with you. However, the problem is not so simple. The main issue is that people can always change an existing specification, which could mean that what you have in the multi-document is perhaps misleading. We should lock specifications actually used in the workspace, which is not so simple. For the next major release...
